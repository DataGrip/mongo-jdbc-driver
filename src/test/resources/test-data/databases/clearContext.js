// command
const v = 1;
// command
clearContext();
// command
v = 3;