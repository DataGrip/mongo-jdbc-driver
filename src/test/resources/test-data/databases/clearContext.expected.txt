No result
No result
|result|
|3|